check_http_json.rb

```diff
@@ -18,23 +18,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 # Requires.
-require 'rubygems'
```

Comment:
This will break compatibility for old Rubies. :cry: Given the, er, *quaint* nature of many a Nagios server out there, I'm hesitant to pull this trigger. Thoughts?